ci/pipeline.yml

```diff
@@ -95,6 +95,20 @@ jobs:
       AWS_DEFAULT_REGION: {{aws-region}}
       GATEWAY_HOST: {{gateway-host}}
 
+- name: prometheus-down-check
+  serial_groups: [production]
+  plan:
+  - aggregate:
+    - get: prometheus-check-timer
+      trigger: true
+    - get: prometheus-config
+      resource: prometheus-config
+  - task: prometheus-down
```

Comment:
Is it possible that this task could fail? We could add on `on_failure` hook to take some action if it does.

Reply:
I'm not sure what "fail" means here.  Like as in can't run?  Or errors out because of lack of memory or remaining processes or something?  I'm not sure what we'd do in that case.  Write a watcher who watches the watcher which is watching the watcher?

Reply:
The script uses `nounset`, so if we forget to set an environment variable, the script can exit before it gets to the end. If emails or pages got sent from the script itself, a typo or unexpected behavior could keep the script from actually checking prometheus. But now that we're sending pages via a hook, any failure will trigger a page, which is perfect.